tox.ini

```diff
@@ -30,6 +30,8 @@ envlist =
     {pypy,py2.7,py3.5,py3.6,py3.7,py3.8,py3.9}-flask-1.1
     {py3.6,py3.8,py3.9}-flask-2.0
 
+    {py3.7,py3.8,py3.9}-quart
```

Comment:
Do we want to do a min quart version we support?

Reply:
Added as 0.16.0